servers/lib/test/testUtil.ts

```diff
@@ -29,7 +29,7 @@ export const fstestFileContent = "content123";
 export const pathToTestFileContent = "user2/tools/README.md";
 export const testFileArray = [
   "data",
-  "digital_twins",
+  "digital twins",
```

Comment:
please keep the underscore

Reply:
Then the repo in Gitlab should be renamed otherwise we have a conflict in the test